code/game/objects/items/gum.dm

```diff
@@ -143,15 +164,20 @@
 	wrapped = FALSE
 	update_icon()
 
+/obj/item/gum/attackby(var/obj/item/W, var/mob/user)
+	if(locked_to)
+		var/datum/locking_category/category = locked_to.get_lock_cat_for(src)
+		if(istype(category, /datum/locking_category/gum_stuck) && is_type_in_list(W, list(/obj/item/weapon/chisel, /obj/item/tool/screwdriver)))
```

Comment:
I'd like to see this with a do_after delay so it takes some time to remove the gum (making it extra annoying)